specreduce/background.py

```diff
@@ -44,8 +44,22 @@ class Background(_ImageParser):
         pixels.
     disp_axis : int
         dispersion axis
+        [default: 1]
     crossdisp_axis : int
         cross-dispersion axis
+        [default: 0]
+    mask_treatment : string, optional
+        The method for handling masked or non-finite data. Choice of `filter`,
```

Comment:
here and elsewhere need to use double single-quotes for inline literal/code. pretty sure this is what RTD has been complaining about...